frida-sys/frida-core.h

```diff
@@ -219,6 +219,7 @@
 #define g_assertion_message_cmpstrv _frida_g_assertion_message_cmpstrv
 #define g_assertion_message_error _frida_g_assertion_message_error
 #define g_assertion_message_expr _frida_g_assertion_message_expr
+#define g_assertion_set_handler _frida_g_assertion_set_handler
```

Comment:
Please uncheck this file from this PR. I updated Frida recently and this appears to be the old one?